dandi/metadata/util.py

```diff
@@ -329,66 +329,68 @@ def extract_cellLine(metadata: dict) -> str | None:
         return None
 
 
+NCBITAXON_URI_TEMPLATE = "http://purl.obolibrary.org/obo/NCBITaxon_{}"
```

Comment:
pushed a renaming for that template variable. If really don't like it -- revert, but I think that it is better when more specific like this